### PR TITLE
Adding linting to sample app (Closes #14)

### LINKED ISSUE #14: Sample app needs npm run lint command to lint HTML, CSS, and JS

### PR DESCRIPTION
**Pull request by alallier**
Closes #14

### CODE REVIEW

File: sampleApp/mvc/views/404.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<include src='layouts/main'>
+<include src="layouts/main">
```
Comment (kethinov) [inline sampleApp/mvc/views/404.html]: we want to keep single quotes on html

**Comment by alallier** [comment]
Updated

**Comment by alallier** [comment]
Updated and removed merge conflict.

**Comment by kethinov** [comment]
Too difficult to resolve the conflict now. Closing this.

**Pull request closed without merging**